### PR TITLE
test: enable goconst linter

### PR DESCRIPTION
**Pull request by llamerada-jp**
issue: #853

### CODE REVIEW

File: test/e2e/clone_test.go [hunk 1]
```diff
@@ -61,15 +61,14 @@ func testPVCClone() {
 		}).Should(Succeed())
 
 		By("writing a file on mountpath")
-		writePath := "/test1/bootstrap.log"
 		Eventually(func() error {
-			_, err := kubectl("exec", "-n", nsCloneTest, "thinpod", "--", "cp", "/var/log/bootstrap.log", writePath)
+			_, err := kubectl("exec", "-n", nsCloneTest, "thinpod", "--", "cp", "/var/log/bootstrap.log", "/test1/bootstrap.log")
```
Comment (jakobmoellerdev) [inline test/e2e/clone_test.go:65]: I think this change is actually bad, and if a linter flags this, we should not adopt it like this.

File: test/e2e/thin_provisioning_test.go [hunk 2]
```diff
@@ -115,9 +110,9 @@ func testThinProvisioning() {
 			var lvName string
 			var err error
 
-			num := strconv.Itoa(i)
+			i := i
```
Comment (jakobmoellerdev) [inline test/e2e/thin_provisioning_test.go:113]: redundant scoped idx

File: test/e2e/thin_provisioning_test.go [hunk 3]
```diff
@@ -127,26 +122,24 @@ func testThinProvisioning() {
...
-			_, err := kubectl("delete", "-n", ns, "pod", "thinpod"+num)
+
+			_, err := kubectl("delete", "-n", ns, "pod", fmt.Sprintf("thinpod%d", i))
 			Expect(err).ShouldNot(HaveOccurred())
-			_, err = kubectl("delete", "-n", ns, "pvc", "thinvol"+num)
+			_, err = kubectl("delete", "-n", ns, "pvc", fmt.Sprintf("thinvol%d", i))
 			Expect(err).ShouldNot(HaveOccurred())
 
 			By("confirming the Pod is deleted")
+			i := i
```
Comment (jakobmoellerdev) [inline test/e2e/thin_provisioning_test.go:139]: redundant scoped idx

File: internal/lvmd/lvservice_test.go [hunk 4]
```diff
@@ -17,6 +17,11 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+const (
```
Comment (jakobmoellerdev) [inline internal/lvmd/lvservice_test.go:20]: these consts should be function scope not global scope

**Review by jakobmoellerdev — CHANGES_REQUESTED** [review]
I have my doubts about this linter implementation. I dont think all the changes here are good for maintenance. Waiting for other opinions.

**Review by cupnes — APPROVED** [review]

**Review by ushitora-anqou — APPROVED** [review]
The current (fixed) version looks good to me.

**Review by jakobmoellerdev — APPROVED** [review]
lgtm

**Pull request merged**